WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -1025,21 +1025,22 @@ extension OrderDetailsDataSource {
 
             var rows: [Row] = Array(repeating: .aggregateOrderItem, count: aggregateOrderItemCount)
 
-            if shouldShowShippingLabelCreation {
+            switch (shouldShowShippingLabelCreation, isProcessingStatus, isRefundedStatus) {
+            case (true, false, false):
+                // Order completed and eligible for shipping label creation:
                 rows.append(.shippingLabelCreateButton)
-            }
-
-            if isProcessingPayment {
-                if shouldShowShippingLabelCreation {
-                    rows.append(.markCompleteButton(style: .secondary, showsBottomSpacing: false))
-                    rows.append(.shippingLabelCreationInfo(showsSeparator: false))
-                } else {
-                    rows.append(.markCompleteButton(style: .primary, showsBottomSpacing: true))
-                }
-            } else if isRefundedStatus == false {
-                if shouldShowShippingLabelCreation {
-                    rows.append(.shippingLabelCreationInfo(showsSeparator: true))
-                }
+                rows.append(.shippingLabelCreationInfo(showsSeparator: false))
```

Comment:
nit: Since both cases now show the shipping label creation info without a separator, and it doesn't look like we use this row elsewhere, we could clean up this row (and related code) to remove the `showsSeparator` parameter and just always set that to `false` when the cell is configured.

I wouldn't consider that a blocker for merging this change, though!

Reply:
Thanks for the review! I'll take a look at these nits separately as I'm EOD and I can fit this one into 9.7 🙇